storage/s3/src/main/java/io/aiven/kafka/tieredstorage/storage/s3/S3StorageConfig.java

```diff
@@ -153,47 +156,58 @@ && getPassword(AWS_ACCESS_KEY_ID_CONFIG) != null) {
         }
     }
 
-    AmazonS3 s3Client() {
-        final AmazonS3ClientBuilder s3ClientBuilder = AmazonS3ClientBuilder.standard();
+    S3Client s3Client() {
+        final S3ClientBuilder s3ClientBuilder = S3Client.builder();
         final String s3ServiceEndpoint = getString(S3_ENDPOINT_URL_CONFIG);
         final String region = getString(S3_REGION_CONFIG);
         if (Objects.isNull(s3ServiceEndpoint)) {
```

Comment:
```suggestion
        if (Objects.isNull(s3ServiceEndpoint())) {
```
I guess? Then the variable can be removed

Reply:
Fixed